spot-webdriver/specs/in-meeting.spec.js

```diff
@@ -0,0 +1,25 @@
+const SpotSession = require('../user/spot-session');
+
+describe('While in a meeting ', () => {
+    const userFactory = require('../user/user-factory');
+    const spotTV = userFactory.getSpotTV();
+    const spotRemote = userFactory.getSpotRemote();
+    const spotSession = new SpotSession(spotTV, spotRemote);
+
+    spotSession.connectRemoteToTV();
+    spotSession.joinMeeting();
+
+    const meetingRemoteControlPage = spotRemote.getInMeetingPage();
+
+    meetingRemoteControlPage.waitForVisible();
+
+    it('can toggle audio mute', () => {
+        meetingRemoteControlPage.muteAudio();
+
+        meetingRemoteControlPage.waitForAudioMutedStateToBe(true);
+
+        meetingRemoteControlPage.unmuteAudio();
+
+        meetingRemoteControlPage.waitForAudioMutedStateToBe(false);
```

Comment:
I wonder if waiting for mute state should be part of the unmute/mute action. The downside I see is tests aren't as explicit about what is being tested.

Reply:
This applies to anything I've already written too.